demos/slack-alert-app/slackbot.py

```diff
@@ -0,0 +1,116 @@
+import time
+import logging
+import os
+import schedule
+
+import pxapi
+from slack.web.client import WebClient
+from slack.errors import SlackApiError
+
+
+# Pixie PxL script.
+PXL_SCRIPT = """
+import px
+
+df = px.DataFrame(table='http_events', start_time='-5m')
+
+# Filter for px-sock-shop namespace only.
+df.namespace = df.ctx['namespace']
+df = df[df.namespace == 'px-sock-shop']
+
+# Add column for service name.
+df.service = df.ctx['service']
+
+# Add olumn for HTTP response status errors.
+df.error = df.http_resp_status >= 400
+
+# Group HTTP events by service, counting errors and total HTTP events.
+df = df.groupby(['service']).agg(
+    error_count=('error', px.sum),
+    total_requests=('http_resp_status', px.count)
+)
+
+px.display(df, "status")
+"""
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+def getPixieData(api_key, cluster_id):
```

Comment:
you'll want to switch over to snake case instead of camel case. Our python standard is to use snake case. 

`getPixieData` would become `get_pixie_data`